README.md

```diff
@@ -229,6 +231,23 @@ The `-throughput -1` specifies the writes tries to write the events at the maxim
 User can use the options `-writecsv  <file name>` to record the latencies of writers and `-readcsv <file name>` for readers.
 in case of End to End latency mode, if the user can supply only `-readcsv` to get the end to end latency in to the csv file.
 
+## Configuring Security Parameters
+
+To run against Security Enabled Pravega, you will need to specify TLS and authentication/authorization parameters
+for the client via the `JAVA_TOOL_OPTIONS` environment variable, before executing this tool.
+
+Below is an example:
+
+```
+JAVA_TOOL_OPTIONS="-Djavax.net.ssl.trustStore=/path/to/client.truststore.jks"
+JAVA_TOOL_OPTIONS="${JAVA_TOOL_OPTIONS} -Dpravega.client.auth.method=Basic"
+JAVA_TOOL_OPTIONS="${JAVA_TOOL_OPTIONS} -Dpravega.client.auth.token=YWRtaW46MTExMV9hYWFh"
```

Comment:
Can you provide some details about how to get this token in a security-enabled Pravega?

Reply:
Alright, done.